resources/devbox/iis-https/server.csr.cnf

```diff
@@ -0,0 +1,13 @@
+[req]
+default_bits = 2048
+prompt = no
+default_md = sha256
+distinguished_name = dn
+
+[dn]
+C=AU
+ST=Tasmania
+L=Hobart
+O=SIL International
+emailAddress=marc_durdin@sil.org
+CN = keyman.com.local
```

Comment:
The setup prompted me for all of these fields, so maybe you can leave your PII off?